src/aws-greengrasstesting-runtime.py

```diff
@@ -0,0 +1,79 @@
+from pytest import fixture
+from pytest import mark
+from src.GGTestUtils import GGTestUtils
+from src.SystemInterface import SystemInterface
+from config import config
+
+
+@fixture(scope="function")    # Runs for each test function
+def gg_util_obj():
+    # Setup an instance of the GGUtils class. It is then passed to the
+    # test functions.
+    gg_util = GGTestUtils(config.aws_account, config.s3_bucket_name,
+                          config.region, config.ggl_cli_bin_path)
```

Comment:
`config.ggl_install_dir` might need to be added. @patrzhan added a commit today morning adding that in.